File: app/src/main/java/forpdateam/ru/forpda/entity/app/history/IHistoryItem.java
package forpdateam.ru.forpda.entity.app.history;

/**
 * Created by radiationx on 01.01.18.
 */

public interface IHistoryItem {
    int getId();

    String getUrl();

    String getDate();

    String getTitle();

    long getUnixTime();

    void setId(int id);

    void setUrl(String url);

    void setDate(String date);

    void setTitle(String title);

    void setUnixTime(long unixTime);
}

File: app/src/main/java/forpdateam/ru/forpda/entity/app/history/IHistoryItem.kt
package forpdateam.ru.forpda.entity.app.history

/**
 * Created by radiationx on 01.01.18.
 */
interface IHistoryItem {
    var id: Int

    var url: String?

    var date: String?

    var title: String?

    var unixTime: Long
}
